### PR TITLE
Renaming TheTest to UserPropertyImplTest for clarity

### PR DESCRIPTION
**Pull request by jglick**

### CODE REVIEW

File: src/test/java/org/jenkinsci/main/modules/cli/auth/ssh/UserPropertyImplTest.java [hunk 1]
```diff
@@ -29,7 +29,6 @@ private void testRoundtrip(String publicKey) throws Exception {
         foo.addProperty(new UserPropertyImpl(publicKey));
         r.configRoundtrip(foo);
         assertEquals(publicKey, foo.getProperty(UserPropertyImpl.class).authorizedKeys);
-        // integration testing w/ sshd-module is in hudson.cli.CLITest.strictHostKey in core
```
Comment (jglick) [inline src/test/java/org/jenkinsci/main/modules/cli/auth/ssh/UserPropertyImplTest.java:32]: Obsolete after #38 now that https://github.com/jenkinsci/sshd-plugin/blob/da71afe504618988831c66063c82687ae481bee4/src/test/java/org/jenkinsci/main/modules/cli/auth/ssh/CLITest.java#L124 is here.

**Review by kuisathaverat — APPROVED** [review]
LGTM

File: src/test/java/org/jenkinsci/main/modules/cli/auth/ssh/UserPropertyImplTest.java (hunk 1 above)
Comment (jglick) [reply src/test/java/org/jenkinsci/main/modules/cli/auth/ssh/UserPropertyImplTest.java:32]: (even after #44)

**Review by oleg-nenashev — APPROVED** [review]
OK?

**Review by timja — APPROVED** [review]

**Pull request merged**